apack.json

```diff
@@ -6,6 +6,7 @@
   "requiredFeatureLevel": "apack.core:9",
   "featureLevel": 106,
   "uc.triggerExtension": "zap",
+  "uc.sdkProvidedProperties": "zcl.matterZclJsonFile,zcl.templateJsonFile",
```

Comment:
What about Zigbee's zcl and templates json files?

Reply:
not sure, have to ask Will